lcdtest.py

```diff
@@ -192,8 +228,43 @@ def options_pop(value, default=no_default):
         pins_data = [int(pin) for pin in pins_data]
         lcd = gpio.CharLCD(pin_rs=rs, pin_rw=rw, pin_e=e, pins_data=pins_data, pin_backlight=bl,
                            numbering_mode=numbering_mode, cols=cols, rows=rows, charmap=charmap)
+    elif lcdmode == 'pigpio':
+
+        from pigpio import pi
+        from RPLCD import pigpio
+
+        if len(sys.argv) < 7:
+            print_usage()
+
+        # pigpio options
+        host = options_pop('host')
+        port = options_pop('port')
+        if host == 'None':
+            pi = pi()
+        elif port == 'None':
+            pi = pi(host)
+        else:
+            pi = pi(host, port)
+
+        data = options_pop('data')
+        rs = int(options_pop('rs'))
+        e = int(options_pop('e'))
+        rw = options_pop('rw', 'None')
+        rw = None if rw == 'None' else int(rw)
+        bl = options_pop('bl', 'None')
+        bl = None if bl == 'None' else int(bl)
+
+        # Parse data pins into a list
+        pins_data = {}
+        pins_data = data.split(',')
+        # Convert data pins to int
+        pins_data = [int(pin) for pin in pins_data]
+        lcd = pigpio.CharLCD(pi,
+                             pin_rs=rs, pin_rw=rw, pin_e=e, pins_data=pins_data, pin_backlight=bl,
+                             numbering_mode=numbering_mode, cols=cols, rows=rows, charmap=charmap)
```

Comment:
You can remove the `numbering_mode` param, it's unused in pgpio mode.

Reply:
You are absolutely right in both cases. Thanks for spotting those mistakes. I just pushed the changes.